README-developers.md

```
@@ -0,0 +1,74 @@
+  * Use the text from `CHANGELOG.md` as the description
+  * Attach the .jar and -all.jar files from `build/libs/`
+  * Click "publish release"
+* Finally, run on the CSE filesystem:  git pull && ./gradlew javadocWeb
+* Update clients and test, so that if it's broken we can re-release.
+
+### Making a snapshot release
+
+* git pull
+* Set version to end in "-SNAPSHOT".
+* Make the snapshot release.
+   * Approach 1:  to Maven Central
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 3

(MD007, ul-indent)